src/cdk/commands/aslVisualizationCDKManager.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { getLogger, Logger } from '../../shared/logger'
+import { AslVisualizationCDK } from './aslVisualizationCDK'
+import { ConstructNode } from '../explorer/nodes/constructNode'
+import { renderGraphCommand } from './renderGraph'
+import { StateMachineGraphCache } from '../../../src/stepFunctions/utils'
+import { AbstractAslVisualizationManager } from '../../../src/stepFunctions/commands/visualizeStateMachine/abstractAslVisualizationManager'
+
+export class AslVisualizationCDKManager extends AbstractAslVisualizationManager {
+
+    public constructor(extensionContext: vscode.ExtensionContext) {
+        super(extensionContext)
+    }
+
+    public async visualizeStateMachine(
+        globalStorage: vscode.Memento,
+        node: ConstructNode
+    ): Promise<vscode.WebviewPanel | undefined> {
+        const logger: Logger = getLogger()
+        const cache = new StateMachineGraphCache()
+        const uniqueIdentifier = node.label
+        const cdkOutPath = node.id?.replace(`/tree.json/${node.tooltip}`, ``)
+        const stackName = node.tooltip?.replace(`/${uniqueIdentifier}`, ``)
+        // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
+        const templatePath = cdkOutPath + `/${stackName}.template.json`
+        const uri = vscode.Uri.file(templatePath);
+        // Attempt to retrieve existing visualization if it exists.
+        const existingVisualization = this.getExistingVisualization(uri.path + uniqueIdentifier)
```

Comment:
It feels a bit clunky to use a key of the template path joined to the unique identifier. I would prefer to use a Map of strings to Map of strings to AslVisualizations. 
```
Map<string, Map<string, AslVisualization>>
```
Since we only maintain this data structure for open visualizations, it shouldn't grow to a size where the difference in memory is significant.